client/src/Controller/Report/ReportController.php

```diff
@@ -42,56 +42,56 @@ class ReportController extends AbstractController
      * @var array
      */
     private static $reportGroupsAll = [
```

Comment:
Changed to alphabetical order 🤩

Reply:
I've started doing this everywhere I see a list of things - major eye twitching otherwise!